akka-http-core/src/main/scala/akka/http/impl/engine/client/OutgoingConnectionBlueprint.scala

```diff
@@ -87,10 +88,12 @@ private[http] object OutgoingConnectionBlueprint {
       val terminationFanout = b.add(Broadcast[HttpResponse](2))
       val terminationMerge = b.add(new TerminationMerge)
 
-      val logger = Flow[ByteString].transform(() ⇒ errorLogger(log, "Outgoing request stream error")).named("errorLogger")
-      val bytesOut = (terminationMerge.out ~> requestRendering.via(logger)).outlet
+      val logger = b.add(Flow[ByteString].transform(() ⇒ errorLogger(log, "Outgoing request stream error")).named("errorLogger"))
+      val wrapTls = b.add(Flow[ByteString].map[SslTlsOutbound](SendBytes))
```

Comment:
Does this not work without the type ascription? That would point to a missing variance annotation somewhere …